yorkie/backend/mongo/client.go

```diff
@@ -311,6 +311,12 @@ func (c *Client) CreateChangeInfos(
 		var modelChanges []mongo.WriteModel
 
 		for _, c := range changes {
+			encodedOperations, err := types.EncodeOperation(c.Operations())
+			if err != nil {
+				log.Logger.Error(err)
```

Comment:
Since log.Logger.Error prints stack trace, it would be better to call only when an error occurs in the method, not our code. Can we move it to below position?
https://github.com/yorkie-team/yorkie/pull/84/files#diff-5a46bd121e6927f28ccfccbafe3ea9ffL46

Reply:
I changed position of `log.Logger.Error(err)`. Is it right?